templates/common/_base/files/configure-ovs-network.yaml

```diff
@@ -425,6 +426,123 @@ contents:
       return $s
     }
 
+    # Accepts parameters $iface_default_hint_file, $iface
+    # Writes content of $iface into $iface_default_hint_file
+    write_iface_default_hint() {
+      local iface_default_hint_file="$1"
+      local iface="$2"
+
+      echo "${iface}" >| "${iface_default_hint_file}"
```

Comment:
(I had never seen `>|` before, we don't have `-o noclobber` here do we?)

Reply:
No we don't, and it's out of precaution and not needed, but it doesn't hurt, either :-) 

Thanks for approving this!